.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,26 +1,47 @@
-<!-- Please replace #XX below with an existing issue number. Remove the line entirely if none exist. -->
-Fixes #XX
+## Fixes
+<!-- If PR only partly solves an issue, replace 'Fixes' below with 'Partially addresses'. -->
+<!-- If there is no issue being resolved, delete the section entirely. -->
+Fixes #[issue number] by @[issue author]
+
+## Type
+<!-- Choose from one of these options -->
+This PR is a [feature|bug-fix|refactor].
```

Comment:
Can't this be implied be the issue type?

Reply:
@brenoferreira not every PR is in response to an issue so I guess it would be helpful in those cases.

Reply:
We ask people to create an issue before sending a PR, it's part of the contribution guidelines.

Reply:
In that case, I'm guilty of opening PRs without issues. I'll remove the section though.

Reply:
Removed by c8a6295.